### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by Kavita539**
Added Quiz For Categories
 Categories:
Stranger things
CAOS

### CODE REVIEW

File: src/pages/explore-quiz/ExploreQuiz.tsx [hunk 1]
```diff
@@ -1,44 +1,49 @@
...
+    } = useQuiz();
+
+    const { quizId } = useParams();
+
+    useEffect(() => {
+    setQuizData(quizzes.filter((quiz: DocumentData) => quiz.quizCategory === quizId));
+    }, [quizId, quizzes]);
+
+    console.log(quizId, "hello");
+    console.log(quizzes);
```
Comment (nerdyspook) [inline src/pages/explore-quiz/ExploreQuiz.tsx]: You can remove the logs from here as well.

**Review by nerdyspook — APPROVED** [review]
Code looks well structured and clean.

File: src/pages/explore-quiz/ExploreQuiz.tsx (hunk 1 above)
Comment (Kavita539) [reply src/pages/explore-quiz/ExploreQuiz.tsx]: Removed it, thanks1

**Review by KanekarSnehal — APPROVED** [review]
Code is clean, you can merge it, no issues found.

**Pull request merged**